<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchJobManDlgClass</class>
 <widget class="QDialog" name="BatchJobManDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BatchJobManDlg</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>361</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>URL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>18</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bjmd_sb_wildcard_begin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wildcard bytes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>http://www.gzl.org/file(*).zip</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bjmd_sb_wildcard_size">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="bjmd_rb_wildcard_digit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>86</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="bjmd_rb_wildcard_char">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bjmd_sb_wildcard_end">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>99</number>
   </property>
  </widget>
  <widget class="QListWidget" name="bjmd_lw_url_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bjmd_le_wildcard_url">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>http://www.qtchina.net/file(*).zip</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>156</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: It is case sensitive</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bjmd_le_wildcard_begin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>a</string>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="bjmd_le_wildcard_end">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>BatchJobManDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>BatchJobManDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bjmd_rb_wildcard_char</sender>
   <signal>clicked(bool)</signal>
   <receiver>bjmd_le_wildcard_begin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bjmd_rb_wildcard_char</sender>
   <signal>clicked(bool)</signal>
   <receiver>bjmd_le_wildcard_end</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bjmd_rb_wildcard_digit</sender>
   <signal>clicked(bool)</signal>
   <receiver>bjmd_le_wildcard_end</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bjmd_rb_wildcard_digit</sender>
   <signal>clicked(bool)</signal>
   <receiver>bjmd_le_wildcard_begin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bjmd_rb_wildcard_char</sender>
   <signal>clicked(bool)</signal>
   <receiver>bjmd_sb_wildcard_begin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bjmd_rb_wildcard_char</sender>
   <signal>clicked(bool)</signal>
   <receiver>bjmd_sb_wildcard_end</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bjmd_rb_wildcard_char</sender>
   <signal>clicked(bool)</signal>
   <receiver>bjmd_sb_wildcard_size</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bjmd_rb_wildcard_digit</sender>
   <signal>clicked(bool)</signal>
   <receiver>bjmd_sb_wildcard_begin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bjmd_rb_wildcard_digit</sender>
   <signal>clicked(bool)</signal>
   <receiver>bjmd_sb_wildcard_end</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bjmd_rb_wildcard_digit</sender>
   <signal>clicked(bool)</signal>
   <receiver>bjmd_sb_wildcard_size</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
